e2e_test/source_inline/tvf/postgres_query.slt

```diff
@@ -1,5 +1,13 @@
+control substitution on
+
+system ok
+source .risingwave/config/risedev-env
```

Comment:
I think this makes sense for now, to declare the variables locally inside this file.

But later we can do something like:
```
export $(cat risedev-env | xargs)
```

In source test, to export all these variables.

Reply:
`risedev slt`  will load this env file, so you don't need to do it here. And it both work in CI and locally.